common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
```

Comment:
@clrux  Could you make it a real template and then render it in this context instead?  That would allow you to use the variables in the template without having it as part of this function's logic.